mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -293,5 +293,20 @@
    */
   public static final String OBJECT_STORE_ELEMENT_NAMESPACE = "os";
 
+  /**
+   * The name of the parameter which configures pooling source item limits per poll.
```

Comment:
typo: polling* and limit*.
Would also maybe rephrase to: The name of the parameter which sets the item limit per poll for polling sources. ("for polling sources" is contextual information that is better to leave for the end of the sentence) ?